src/Sarif.Viewer.VisualStudio/CodeLocationObject.cs

```diff
@@ -94,7 +106,7 @@ internal virtual string UriBaseId
             }
             set
             {
-                if (string.Equals(value, this._uriBaseId))
+                if (!string.Equals(value, this._uriBaseId))
```

Comment:
>! [](start = 20, length = 1)

This backwards test prevented `CodeLocationObject.UriBaseId` from being set, and that _is_ a regression. But this isn't the `UriBaseId` whose null-ness caused the bug.